products/zoningtaxlots/models/intermediate/int__zoningtaxlots.sql

```diff
@@ -179,34 +144,16 @@ add_zoningmapnumber AS (
         a.specialdistrict3,
         a.limitedheightdistrict,
         (CASE
-            WHEN b.perbblgeom >= 10 THEN b.zoning_map
-        END) AS zoningmapnumber
-    FROM add_height AS a
-    LEFT JOIN zoningmapindex AS b
-        ON a.dtm_id = b.dtm_id AND b.row_number = 1
-),
-
-add_zoningmapcode AS (
-    SELECT
-        a.dtm_id,
-        a.bbl,
-        a.boroughcode,
-        a.taxblock,
-        a.taxlot,
-        a.area,
-        a.commercialoverlay1,
-        a.commercialoverlay2,
-        a.specialdistrict1,
-        a.specialdistrict2,
-        a.specialdistrict3,
-        a.limitedheightdistrict,
-        a.zoningmapnumber,
+            WHEN b1.perbblgeom >= 10 THEN b1.zoning_map
+        END) AS zoningmapnumber,
```

Comment:
Yeah this seems like the one where it's fine to be leaving this bit of logic here, since it's a tad funky (and just different enough from a super simple pivot)